std_msgs/Header header
nav2d_msgs/LocalizedScan[] data
int32[] secs
int32 alert_flag